packages/react-i18n/src/tests/i18n.test.ts

```diff
@@ -718,12 +718,13 @@ describe('I18n', () => {
         expect(i18n.unformatNumber('1.234.567,56')).toBe('1234567.56');
       });
 
-      it('does not treat . as the decimal symbol if , is not used as a decimal symbol', () => {
+      it('treats . as the decimal symbol if . is used as a decimal symbol', () => {
```

Comment:
I don't have a lot of context on react-i18n, so I just want to make sure I understand what we are solving.

The tests cases that we are changing here were actually covered before but the assumption was that the result should be different?
So for example. with the previous changes we thought that `unformatNumber('1234.50') === 123450` but in order to make it backwards compatible we need it to be `unformatNumber('1234.50') === 1234.5` instead?

Reply:
Yes; previously I was making the assumption that `unformatCurrency('1234.50', 'EUR')` should be formatted as `123450.00`, but that's because I wanted to support the Vietnamese locale and I didn't think of a better way to do that while remaining backwards compatible.

So I'm essentially restoring the functionality I [changed in my previous PR](https://github.com/Shopify/quilt/pull/2126/files#diff-a745a62ae45742714f76cc54da399803f886c30142880938f387687063fc7c85L1169), while also being compatible with Vietnamese number formatting (e.g. `unformatCurrency('1.234', 'VND') == '1234.00'`)